server/fleet/hosts.go

```diff
@@ -316,6 +316,9 @@ type Host struct {
 	// The boolean is based on information ingested from the Apple DEP API that is stored in the
 	// host_dep_assignments table.
 	DEPAssignedToFleet *bool `json:"dep_assigned_to_fleet,omitempty" db:"dep_assigned_to_fleet" csv:"-"`
+
+	// LastRestarted is a UNIX timestamp that indicates when the Host was last restarted.
```

Comment:
Related to https://github.com/fleetdm/fleet/pull/14878/files#r1380221391 maybe representing `last_restarted` as a date (similar to the UI) also works for sorting?

Reply:
Yeah I think that doing a date will work for sorting and then we can have an empty string in JSON representations.

Reply:
Now that we are returning this field, do we need `humanHostLastRestart` in the UI?

Reply:
Don't think so, removed it @lucasmrod !